packages/flutter_test/lib/src/test_default_binary_messenger.dart

```diff
@@ -116,11 +121,17 @@ class TestDefaultBinaryMessenger extends BinaryMessenger {
   // can implement the [checkMockMessageHandler] method.
   final Map<String, Object> _outboundHandlerIdentities = <String, Object>{};
 
+  /// Handler that intercepts and responds to outgoing messages, pretending
+  /// to be the platform, for all channels.
+  AllMessagesHandler? allMessagesHandler;
```

Comment:
setMockMessageHandler documents that "Registered callbacks are cleared after each test.". This one should probably as well, if it isn't already.

Reply:
I couldn't find any references of setMockMessageHandler (or references of references which) in the flutter repo that did that. Any suggestions on where this might be happening?